BlockServer/fileIO/schema_checker.py

```diff
@@ -104,22 +104,36 @@ def check_config_file_matches_schema(schema_folder, config_xml_path, is_componen
         return True
 
     @staticmethod
-    def check_synoptic_matches_schema(schema_filepath, synoptic_xml_data):
+    def check_xml_data_matches_schema(schema_filepath,xml_data):
         """ This method takes xml data and checks it against a given schema.
 
         A ConfigurationInvalidUnderSchema error is raised if the file is incorrect.
 
         Args:
             schema_filepath (string): The location of the schema file
-            synoptic_xml_data (string): The XML for the synoptic
+            synoptic_xml_data (string): The XML for the screens
         """
         folder, file_name = string.rsplit(schema_filepath, os.sep, 1)
         xmlparser = ConfigurationSchemaChecker._import_schema(folder, file_name)
 
         try:
-            etree.fromstring(synoptic_xml_data, xmlparser)
+            etree.fromstring(xml_data, xmlparser)
         except etree.XMLSyntaxError as err:
-            raise ConfigurationInvalidUnderSchema("Synoptic incorrectly formatted: " + str(err.message))
+            raise ConfigurationInvalidUnderSchema(str(err.message))
+
```

Comment:
This is the same as  check_screens_match_schema generalise error message and combine